CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ##### Enhancements
 
+* [8067](https://github.com/grafana/loki/pull/8067) **DylanGuedes**: Distributor: Add auto-forget unhealthy members support.
```

Comment:
```suggestion
* [8067](https://github.com/grafana/loki/pull/8067) **DylanGuedes**: Distributor: Auto-forget unhealthy ring members to mitigate a problem that can lead to `empty ring` error responses.
```

Reply:
Although I previously stated that this should help with the empty ring errors, after testing and better investigation I found that this won't help at all. This will be helpful though in any scenarios where a OOM happens.